jujugui/static/gui/src/app/assets/css/_components.scss

```diff
@@ -107,6 +107,7 @@
 @import '../../components/popup/popup';
 @import '../../components/post-deployment/post-deployment';
 @import '../../components/profile/profile';
+@import '../../components/profile/expanded-content/expanded-content';
```

Comment:
Put this in alphabetical order?

Reply:
It looks like we use a pattern of sub-dirs after files, so this is at least consistent.